android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -115,11 +115,17 @@ public void join(@Nullable String url) {
     }
 
     public void join(JitsiMeetConferenceOptions options) {
-        getJitsiView().join(options);
+        JitsiMeetView view = getJitsiView();
+        if (view != null) {
```

Comment:
I find it weird to silently not take action on method call. Where does the crash occur and why is it a problem?

Reply:
Since they are public methods they can be called whenever, inside onDestroy too for example. On Our (8x8) case, leave was being called before join because of an external action.

While it may be a bit of a stretch, I thought we'd try to be nice and not crash.

Reply:
Can the `onDestroy` do this check instead? Those are public methods which are public SDP API and my problem is that how user can use them without knowing if the call took any effect?

Reply:
Actually this example of calling leave before join is a good example on why this crash needs to happen. The user app needs to be aware that is doing things wrong.

Reply:
I don't think so, since it can be also overriden.

Reply:
Let me think about some other way to accomplish this.

Reply:
https://docs.oracle.com/javase/tutorial/java/IandI/final.html

Reply:
you may find the link helpful should you decide to block overriding  any of the API methods